judgels-frontends/raphael/src/routes/uriel/contests/ContestsPage/ContestsPage.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../../styles/base';
+
+.search-box-inline {
```

Comment:
Let's add `margin-bottom: 0` to `.search-box-inline` and `.create-contest-button-inline` to reduce the distance between header and the contests list (right now it is too far)